dev/src/setup/dapp.ts

```diff
@@ -9,13 +10,13 @@ export async function setupDapp(env: ProsopoEnvironment, dapp: IDappAccount): Pr
         const tasks = new Tasks(env)
 
         try {
-            await tasks.contractApi.getDappDetails(dapp.contractAccount)
+            ;(await tasks.contract.query.getDappDetails(dapp.contractAccount)).value.unwrap().unwrap()
```

Comment:
defensive semi colon as first line in try block? not sure i follow what that syntax does :)